### PR TITLE
Function documentation cleanup

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/binary.rst [hunk 1]
```diff
@@ -37,77 +53,76 @@ Binary Functions
...
 
 .. function:: from_base64(string) -> varbinary
 
     Decodes binary data from the base64 encoded ``string``.
 
-.. function:: to_base64url(binary) -> varchar
+.. function:: to_base64(binary) -> varchar
 
-    Encodes ``binary`` into a base64 string representation using the URL safe alphabet.
+    Encodes ``binary`` into a base64 string representation.
```
Comment (dain) [inline presto-docs/src/main/sphinx/functions/binary.rst:65]: Should we mention the extra characters or link to an RFC?

File: presto-docs/src/main/sphinx/functions/binary.rst [hunk 2]
```diff
@@ -37,77 +53,76 @@ Binary Functions
...
 .. function:: to_big_endian_32(integer) -> varbinary
 
-    Encodes ``integer`` in a 32-bit 2's complement big endian format.
+    Encodes ``integer`` into a 32-bit two's complement big-endian format.
 
-.. function:: from_big_endian_32(binary) -> integer
+.. function:: from_big_endian_64(binary) -> bigint
 
-    Decodes ``integer`` value from a 32-bit 2's complement big endian ``binary``.
+    Decodes the 64-bit two's complement big-endian ``binary``.
```
Comment (dain) [inline presto-docs/src/main/sphinx/functions/binary.rst:99]: These (int, long, float, double) should mention what happens when binary is greater or less than 4 or 8 bytes

File: presto-docs/src/main/sphinx/functions/binary.rst [hunk 3]
```diff
@@ -142,18 +153,25 @@ Binary Functions
 
     Computes the 64-bit SpookyHashV2 hash of ``binary``.
 
+.. function:: xxhash64(binary) -> varbinary
+
+    Computes the xxhash64 hash of ``binary``.
```
Comment (dain) [inline presto-docs/src/main/sphinx/functions/binary.rst:158]: I believe it should be `xxHash64`

Comment (electrum) [reply presto-docs/src/main/sphinx/functions/binary.rst:158]: There doesn't seem to be consensus, even from the author, but I agree that the capitalized version looks nicer.

**Pull request closed without merging**